seed/seed.json

```diff
@@ -685,6 +685,33 @@
                 "channel": ["RELEASE", "BETA", "NIGHTLY"],
                 "platform": ["MAC"]
             }
+        },
+        {
+            "name": "BraveTalk",
```

Comment:
I think having `Study` suffix in name like `BraveTalkStudy` is its convention.
@goodov Am I right? :)

Reply:
Good catch! Fixed 😄 👍